use-cases/call-center/src/test/java/org/acme/callcenter/solver/SolverServiceTest.java

```diff
@@ -77,6 +76,7 @@ void prolongCall() {
 
     @Test
     @Timeout(60)
+    @Retry(3)
```

Comment:
While this approach lowers the risk of the test failing again, it does not fix the root cause.

This is a not a problem with a random failure due to e.g. network latency, which would justify the retries.

Suggestion: try reproducing the issue by running the test in a loop (perhaps with some very short random pauses) and see if it shows up.